analog_processor:
  ros__parameters:
    board_id: 1
    pins:
      0: 'temp'
      1: 'rh'
      2: 's1'
      3: 's2'
      4: 's3'
      5: 'ground'
    voltage_offset: 0.0
    update_rate: 4.0   # Update frequency in Hz (2.0 = twice per second)
    moving_buffer: 35  # Number of points to include in moving buffer
    trimmean_percent: 50  # Percent of points to take from middle of moving buffer
    # board1[slope,intercept], board2[slope,intercept], board3[slope,intercept]
    temp_coefficients: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
    rh_coefficients: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
